project/src/test/test-player-save-upgrader.gd

```diff
@@ -89,37 +84,17 @@ func test_245b() -> void:
 	load_legacy_player_data("turbofat-245b.json")
 	
 	# some levels were made much harder/different, and their scores should be invalidated
-	assert_true(PlayerData.level_history.level_names().has("marsh/pulling_for_everyone"))
 	assert_false(PlayerData.level_history.level_names().has("marsh/hello_everyone"))
 	assert_false(PlayerData.level_history.level_names().has("marsh/hello_skins"))
 	assert_false(PlayerData.level_history.level_names().has("marsh/pulling_for_skins"))
 	assert_false(PlayerData.level_history.level_names().has("marsh/goodbye_skins"))
 
 
-func test_24cc() -> void:
-	load_legacy_player_data("turbofat-24cc.json")
-	
-	assert_eq(PlayerData.chat_history.chat_history.get("chat/level_select"), 10)
-	assert_eq(PlayerData.chat_history.chat_history.get("creature/bort/filler"), 6)
-	assert_eq(PlayerData.chat_history.chat_counts.get("chat"), 77)
-	assert_eq(PlayerData.chat_history.chat_counts.get("creature/bort"), 24)
-
-
+## Note: There is no test for save data version 24cc. The updater performs some chat history renames, but these chat
+## items are nonexistent in the newest version of the game because of the removal of free roam mode
```

Comment:
Move this up and change it from two pound signs to one. It's misleading to have it attached to `test_2743`

Reply:
Moved